### PR TITLE
🚧 Enhance: Avoid Conflict Helper Function In Some Packages

### PR DESCRIPTION
**Pull request by jalmatari**

### CODE REVIEW

File: src/ScoutElasticSearchServiceProvider.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ public function boot(): void
     {
         $this->loadTranslationsFrom(__DIR__.'/../resources/lang', 'scout');
 
-        resolve(EngineManager::class)->extend(ElasticSearchEngine::class, function () {
+        $this->app->make(EngineManager::class)->extend(ElasticSearchEngine::class, function () {
             $elasticsearch = resolve(Client::class);
```
Comment (matchish) [inline src/ScoutElasticSearchServiceProvider.php:27]: What about this line?

Comment (jalmatari) [reply src/ScoutElasticSearchServiceProvider.php]: Enhanced👌🏻

**Comment by matchish** [comment]
Bump the version plz
And update changelog

**Comment by matchish** [comment]
Thanks for your contribution)

**Pull request merged**